tabpy-tools/tabpy_tools/schema.py

```diff
@@ -97,9 +100,11 @@ def generate_schema(input, output, input_description=None, output_description=No
                    'properties': {'y': {'type': 'integer', 'description': 'value of y'},
                                   'x': {'type': 'integer', 'description': 'value of x'}}},
          'output': {'type': 'integer', 'description': 'x times y'}}
-    '''
-    input_schema = _generate_schema_from_example_and_description(input, input_description)
-    output_schema = _generate_schema_from_example_and_description(output, output_description)
+    '''  # noqa: E501
```

Comment:
Added noqa for now, but may want to look at something like the ipython directive which I think will actually render code for you